masq_lib/src/shared_schema.rs

```diff
@@ -526,9 +526,110 @@ impl ConfiguratorError {
 #[cfg(test)]
 mod tests {
 
+    use super::*;
     use crate::blockchains::chains::Chain;
     use crate::shared_schema::{common_validators, official_chain_names};
 
+    #[test]
+    fn constants_have_correct_values() {
```

Comment:
If you have time and determination you can read through all this tested text and see if there are any remarkable typos or stylistic imperfections etc. If so then its good to take this opportunity to make it right when we put these tests in.